uefi-services/src/lib.rs

```diff
@@ -95,6 +96,51 @@ pub fn init(st: &mut SystemTable<Boot>) -> Result {
     }
 }
 
+// Internal function for print macros
+#[doc(hidden)]
+pub fn _print(args: core::fmt::Arguments) {
+    unsafe {
+        if let Some(st) = &mut SYSTEM_TABLE {
+            st.stdout().write_fmt(args).expect("Failed to write to stdout");
+        } else {
+            panic!("SYSTEM_TABLE is None");
+        }
+    }
+}
+
+/// Prints to the standard output
+/// 
+/// # Panics
+/// Will panic if SYSTEM_TABLE is None (Before [init] and after [exit_boot_services])
```

Comment:
For consistency, could you write the constants as `` `SYSTEM_TABLE` `` and `` `None` ``?

Reply:
👍